docs/docs/keymaps/input-processors/behaviors.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Behaviors Input Processor
+sidebar_label: Behaviors
+---
+
+## Overview
+
+The behaviors input processor is used invoke standard behaviors when certain input events occur; most frequently this is used to trigger behaviors when certain mouse buttons are triggered by physical pointing devices.
+
+## Usage
+
+When used, this input processor takes no parameters, as the event code to behavior mapping is specified in the definition of the specific processor instance, e.g.:
+
+```dts
+&zip_button_behaviors
+```
+
+## Pre-Defined Instances
+
+One pre-defined instance of the out-of-band behaviors input processor is available:
+
+| Reference               | Description                                        |
+| ----------------------- | -------------------------------------------------- |
+| `&zip_button_behaviors` | Maps left/right/middle clicks to a given behavior. |
+
+Should you wish to update the existing instance to trigger different behaviors for each mouse button, you can override the `bindings` property, e.g.:
+
+```dts
+&zip_button_behaviors {
+    bindings = <&kp A &kp B &kp C>;
+};
+```
+
+## User-Defined Instances
+
+Users can define new instances of the out-of-band behaviors input processor if they want to target different codes or assign different behaviors.
+
+### Example
+
+Below example maps the left mouse button code to the middle mouse button.
+
+```dts
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+    input_processors {
+        zip_right_click_trigger_paste: zip_right_click_trigger_paste {
+            compatible = "zmk,input-processor-behaviors";
+            #input-processor-cells = <0>;
+            codes    = <INPUT_BTN_1>;
+            bindings = <&kp LC(V)  >;
+        };
+    };
+}
+```
+
```

Comment:
Can we add an example that uses one of the movement events? Something like "convert pointer movement to arrow keys"? That's one thing that will probably commonly come up as a usage.

As an aside, there might also be requests to convert a certain amount of movement to `n` taps of a key. I can see how that is much more complicated to implement, but mmaybe something to think about.

Reply:
Maybe it is worth at least implementing a min threshold on the accumulated value before a press is triggered? I don't know enough about the input processors to say whether that is easy or not 😅

Reply:
Honestly... At this point this realistically only supports "key" type events with on/off values, not movement events. To your point, we could have some "limit" for values, use the existing remainder tracking to handle this? We'd need special casing for "key" events though, since they should only trigger the behavior release once the same key code is released. Pondering this.

Reply:
This was due to my confusion thinking it applied to movement type events as well -- some discussion followed in https://discord.com/channels/719497620560543766/845285481888743434/1318326844935901215. I think the outcome is that we'll document this limitation for now.

Reply:
@caksoylar I added a more prominent note to the top of the behaviors input processor to explain the limitation. Thoughts?